packages/design-system/src/components/Tooltip/Tooltip.test.jsx

```diff
@@ -40,12 +41,30 @@ describe('Tooltip', function () {
   it('renders dialog tooltip', () => {
     renderTooltip({ dialog: true });
     const tooltipTrigger = screen.getByLabelText(triggerAriaLabelText);
-    fireEvent.click(tooltipTrigger);
+    userEvent.click(tooltipTrigger);
     const contentEl = screen.queryByRole('dialog');
     expect(contentEl).not.toBeNull();
     expect(contentEl).toMatchSnapshot();
   });
 
+  it('closes tooltip when trigger focus is lost', async () => {
```

Comment:
Nice test! I'm glad we're using RTL now. Trying to mock this behavior with Enzyme would have been clunky and fragile